migration/v0.31/apply/12-remove-no-ttl-jobs.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+find_delete_no_ttl_jobs() {
+  for n in $(kubectl_do "${1}" get ns --selector owner=operator -o name); do
```

Comment:
Why selecting only `owner=operator` namespaces?

Reply:
I had the impression that we want this only for our jobs. If not I will change it.

Reply:
I so no reason to not remove all offending jobs.

Reply:
done. in your example you were also taking into consideration the job_start_epoch. let me know if you still think that is necessary

Reply:
I would say it makes sense to check that as well, to withhold the 7 day for jobs if you were to let's say upgrade apps in a quick successsion.